README.md

```diff
@@ -39,7 +39,8 @@ Then commit the changes to `go.mod` and `go.sum`.
 terraform {
   required_providers {
     astro = {
-      source = "registry.terraform.io/astronomer/astro"
+      source = "astronomer/astro"
+      version >= "0.0.0"
```

Comment:
Curious - Why the addition here?

Reply:
oh - true, i thought you always need to specify a version, but you actually don't
thanks!